static/app/views/performance/landing/widgets/components/queryHandler.tsx

```diff
@@ -97,7 +97,8 @@ function QueryResultSaver<T extends WidgetDataConstraint>(
   useEffect(() => {
     const isMetricsData =
       results?.seriesAdditionalInfo?.[props.queryProps.fields[0]]?.isMetricsData ??
-      results?.histograms?.meta?.isMetricsData;
+      results?.histograms?.meta?.isMetricsData ??
+      results?.tableData?.meta?.isMetricsData;
```

Comment:
Added this line to fix the `widgetContainer` test, which kept failing after `CompactSelect` update ([see here](https://github.com/getsentry/sentry/actions/runs/3237838494/jobs/5305379588) – the failure doesn't seem to be a result of the update itself). 

This works because the `results` object may contain a `tableData` key which also has an `isMetricsData` value that we're looking for:

<img width="475" alt="Screen Shot 2022-10-12 at 2 31 26 PM" src="https://user-images.githubusercontent.com/44172267/195452133-abdb08c7-58df-4ab5-8e88-54c7fc8751d5.png">

Reply:
@k-fish 👆

Reply:
sorry, was away, yeah 👍